package.nls.json

```diff
@@ -71,6 +71,7 @@
   "configuration.SAS.results.html.style": "Specifies the style for ODS HTML5 results.",
   "configuration.SAS.results.html.style.(auto)": "Let the extension pick a style that most closely matches the color theme.",
   "configuration.SAS.results.html.style.(server default)": "Default to the style configured on the SAS server.",
+  "configuration.SAS.results.html.custom.style": "Map editor color themes to ODS HTML5 styles. Specify an object with properties like 'light', 'dark', 'highContrast', and 'highContrastLight' to control which style is used for each theme.",
```

Comment:
I'm just thinking shall we mention something like "Takes effect when `SAS.results.html.style` is `(auto)`" here.
If the sentence becomes longer, perhaps the mention of property names can be removed. The schema in the `package.json` file will produce autocomplete when user editing the setting.